portfolio/src/main/webapp/index.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
-<html>
-  <link href="https://fonts.googleapis.com/css2?family=Cookie&display=swap" rel="stylesheet">
+<html lang="en">
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="photo-frame.css">
+    <link href="https://fonts.googleapis.com/css2?family=Cookie&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="default.css">
     <link rel="stylesheet" href="buttons.css">
   </head>
   <body>
-    <div id="greeting-container">
+    <div id="navbar">
       <div>
-        <a class="btn" href="index.html">My hobbies</a>
+        <a class="button" href="/about-me/about-me.html">About me</a>
```

Comment:
If you used a relative path instead of an absolute one, there would be more flexibility regarding how and where the page could be served.

If you decide to go for absolute paths, feel free to do so, but in that case please keep the links consistent and use absolute links everywhere.

Reply:
Done.